src/components/progress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-progress-bar',
+  styleUrl: 'progress-bar.scss',
+  shadow: true
+})
+export class ProgressBar {
+  @Prop() percentage: number = 0;
+  @Prop() label: string = '';
+  @Prop() size: string;
+  @Prop() showLabel: boolean = false;
+
+
```

Comment:
The progress bar value is not emitted as a custom event. Users have no way of capturing the value. 
Add an event emitter, and emit the `this.percentange`. 
Examaple: 
```
  @Event() valueChanged: EventEmitter<number>;

  emitValue() {
    this.valueChanged.emit(this.percentage);
  }
```

Reply:
totally! just looked into it again and gonna do a new commit in a bit with all the improvements